integration-test/src/test/java/com/datastrato/gravitino/integration/test/authorization/ranger/RangerRef.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
```

Comment:
Can the the license pass CI check?

Reply:
No it will not we need to use the ASF license now

Reply:
Fixed